src/renderer.zig

```diff
@@ -188,6 +189,10 @@ pub fn renderWorld(world: *World, ambientLight: Vec3f, skyColor: Vec3f, playerPo
 	Skybox.render();
 	gpu_performance_measuring.stopQuery();
 
+	gpu_performance_measuring.startQuery(.particle_rendering);
+	particles.ParticleManager.render(playerPos, ambientLight);
```

Comment:
Particles should be rendered after opaque chunks to prevent lag spikes from the occlusion culling system.